app/controllers/api/v1/stats/channels_controller.rb

```diff
@@ -0,0 +1,64 @@
+class Api::V1::Stats::ChannelsController < Api::BaseController
+
+  def channels
+
+    if params[:uuid].nil?
```

Comment:
Let's make this RESTful by doing:

```
def index
   render(json: channels.pluck(:id).to_json, status: :ok) and return
end
```

and 

```
def show
  # your other code
end
```

Reply:
Fixed